### ISSUE TITLE
Base package doesn't pass pre-commit

### ISSUE THREAD

**Issue by john-sandall** [CoefficientSystems/coefficient-cookiecutter#23]
```
pylint..........................................................................Failed
- hook id: pylint
- exit code: 16

************* Module tests.test_gershwin
tests/test_gershwin.py:1:0: C0114: Missing module docstring (missing-module-docstring)
tests/test_gershwin.py:6:0: C0116: Missing function or method docstring (missing-function-docstring)

------------------------------------------------------------------
Your code has been rated at 6.67/10 (previous run: 6.67/10, +0.00)
```

## Todo
- [ ] Add module docstring to tests

**Comment by john-sandall** [comment]
Partially solved by #27, it would be good to have a test suite on the cookiecutter that installs the package & runs the internal test suite. Keeping this task open to prompt that.

**Comment by john-sandall** [comment]
This will be resolved by #55
